qiita_pet/templates/study_description_templates/processed_data_info_tab.html

```diff
@@ -14,4 +14,14 @@
       </tr>
     {% end %}
   </table>
+  {% if btn_to_show == 'request_approval' %}
+    <a class="btn btn-default glyphicon glyphicon-eye-open" onClick="request_approval({{pd_id}});" style="word-spacing: -10px;"> Request Approval</a>
```

Comment:
Can you be more specific, users will not know what approval means? Perhaps, "Request making this data public". Same to the other buttons.